packages/backend/src/models/DashboardModel/PersonalAccessTokenModel.ts

```diff
@@ -39,10 +43,67 @@ export class PersonalAccessTokenModel {
         );
     }
 
+    async getUserToken({
+        userUuid,
+        tokenUuid,
+    }: {
+        userUuid: string;
+        tokenUuid: string;
+    }): Promise<PersonalAccessToken> {
+        const row = await this.database('personal_access_tokens')
```

Comment:
should we use `PersonalAccessTokenTableName` here ? we normally use constants on talbes, I think the only benefit is to avoid typos.